Graffle.FlowEventProcessor/Program.cs

```diff
@@ -0,0 +1,157 @@
+                            currentBlockHeight = lastBlockHeight + maximumBlockScanRange;
+                            Console.WriteLine("Crunching....");
+                        }
+
+                        //We have new blocks!
+                        Console.WriteLine($" {DateTimeOffset.UtcNow} - Last Block: {lastBlockHeight++}, Current Block: {currentBlockHeight}");
+
+                        var eventRetryPolicy = Policy
+                        .Handle<Exception>()
+                        .WaitAndRetryAsync(new[]
+                            {
+                                TimeSpan.FromMilliseconds(250),
+                                TimeSpan.FromMilliseconds(500),
+                                TimeSpan.FromMilliseconds(2000)
+                            }, (exception, timeSpan, retryCount, context) =>
+                            {
+                                Console.WriteLine($"Encountered exception getting events for block." + 
+                                $" Retry count: {retryCount}, {timeSpan}: {exception.Message} {exception.InnerException?.Message}");
+                            });
+
+                        var eventsResponse = (await eventRetryPolicy.ExecuteAsync<List<Graffle.FlowSdk.Services.Models.FlowEvent>>(
+                                () => flowClient.GetEventsForHeightRangeAsync(eventId, lastBlockHeight, currentBlockHeight)
+                            )).GroupBy(x => x.BlockHeight).ToDictionary(x => x.Key, x => x.ToList());
+
+                        var totalEventsFound = eventsResponse.Select(x => x.Value.Count).Sum();
+                        var blocksScanned = eventsResponse.Count;
+
+                        Console.WriteLine($" {DateTimeOffset.UtcNow} - Found {totalEventsFound} events across {blocksScanned} blocks.");
+
+                        foreach(var blockHeightGrouping in eventsResponse) 
+                        {
+                            foreach (var @event in blockHeightGrouping.Value)
+                            {
+                                //Create the event
+                                var flowEvent = new ProcessedFlowEvent()
+                                {
+                                    BlockEventData = @event.EventComposite.Data,
+                                    BlockHeight = blockHeightGrouping.Key,
+                                    EventDate = @event.BlockTimestamp,
+                                    FlowBlockId = @event.BlockIdHash,
+                                    FlowEventId = eventId,
+                                    FlowTransactionId = @event.TransactionId.ToHash()
+                                };
+
+                                //Send webhook
+                                var jsonMesage = System.Text.Json.JsonSerializer.Serialize(flowEvent);
+                                using (var content = new StringContent(jsonMesage)) {
+                                    using (var client = new HttpClient()) {
```

Comment:
I would create a HttpClient singleton at start up. Going to get pretty wasteful on high throughput projects